pipeline/docs/ess-dive-submissions.md

```diff
@@ -0,0 +1,38 @@
+
+# Steps to update ESS-DIVE submissions
+
+0.  Run processing pipeline to produce L1 data for synoptic and TEMPEST sites
+
+1.  Move L1 data to folders labeled `vX-X` \> `vX-X_SYN` and `vX-X` \> `vX-X_TMP` . Each folder gets a copy of the `READMD.txt` and `README_vX-X.txt` files.
+
+2.  Prep package metadata files
+
+-   Produce flmd using `flmd-generator.R`
+-   Copy over `dd.csv` from previous version (this file does not change unless we change or add a new L1 column)
+-   Run `availability_graph.R` to produce availability graphs.
+    -   Add saved graphs to quick start documents located in the FME Data Management folder
```

Comment:
Do you create a new copy of the documents for each release?

Reply:
Yes